[2025-04-09T16:39:48.238+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T16:39:48.408+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.load_task scheduled__2025-04-06T00:00:00+00:00 [queued]>
[2025-04-09T16:39:48.448+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.load_task scheduled__2025-04-06T00:00:00+00:00 [queued]>
[2025-04-09T16:39:48.460+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 4
[2025-04-09T16:39:48.671+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load_task> on 2025-04-06 00:00:00+00:00
[2025-04-09T16:39:48.719+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=182) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-09T16:39:48.709+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'load_task', 'scheduled__2025-04-06T00:00:00+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpaw6orx91']
[2025-04-09T16:39:48.756+0000] {standard_task_runner.py:72} INFO - Started process 184 to run task
[2025-04-09T16:39:48.774+0000] {standard_task_runner.py:105} INFO - Job 67: Subtask load_task
[2025-04-09T16:39:49.113+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.load_task scheduled__2025-04-06T00:00:00+00:00 [running]> on host e5bea2914fd7
[2025-04-09T16:39:49.655+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='load_task' AIRFLOW_CTX_EXECUTION_DATE='2025-04-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-06T00:00:00+00:00'
[2025-04-09T16:39:49.675+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-09T16:39:49.879+0000] {etl_pipeline.py:45} INFO - Données à charger : [['2025-04-09', 7.2, 18.9, 0.0, 1], ['2025-04-10', 5.2, 17.3, 0.0, 3], ['2025-04-11', 6.8, 21.1, 0.0, 0], ['2025-04-12', 9.1, 23.2, 1.2, 61], ['2025-04-13', 13.8, 18.7, 0.3, 3], ['2025-04-14', 9.7, 18.1, 0.0, 3], ['2025-04-15', 8.9, 17.8, 1.2, 61]]
[2025-04-09T16:39:49.898+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-09T16:39:50.069+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-09T16:39:50.089+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=load_task, run_id=scheduled__2025-04-06T00:00:00+00:00, execution_date=20250406T000000, start_date=20250409T163948, end_date=20250409T163950
[2025-04-09T16:39:50.257+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T16:39:50.316+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-09T16:39:50.323+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
